### PR TITLE
add docs for item flags / status output, fixes #402

### LINKED ISSUE #402: create -v documentation
I am unsure what the prefix mean:

M - modified
U - unchanged
d - directory
A -?

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -411,6 +411,39 @@ Additional Notes
 
 Here are misc. notes about topics that are maybe not covered in enough detail in the usage section.
 
+Item flags
```
Comment (anarcat) [inline docs/usage.rst]: doesn't this belong to the create manpage?

Comment (ThomasWaldmann) [inline docs/usage.rst]: Yes, it is for create, but I thought it is better in the separate section to not make the create section too long.

Comment (anarcat) [inline docs/usage.rst]: well, create is a separate page and i find the main usage.rst to be a little too long now...

Comment (ThomasWaldmann) [inline docs/usage.rst]: having all in one piece is good for searching.

**Pull request merged**